velox/exec/fuzzer/JoinFuzzer.cpp

```diff
@@ -299,6 +303,27 @@ class JoinFuzzer {
 
   VectorFuzzer vectorFuzzer_;
   std::unique_ptr<ReferenceQueryRunner> referenceQueryRunner_;
+
+  struct Stats {
+    // The total number of iterations tested.
+    size_t totalIterations{0};
+
+    // The number of iterations verified against reference DB.
+    size_t numVerified{0};
+
+    // The number of iterations that test cross product.
+    size_t numCrossProduct{0};
+
+    void print() const {
```

Comment:
The common practice is to provide a toString method and let the control path to log out if need.